tutorial/advanced/t12_hyperparameter_search.ipynb

```diff
@@ -0,0 +1,676 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Advanced Tutorial 12: Hyperparameter Search\n",
+    "\n",
+    "## Overview\n",
+    "In this tutorial, we will discuss the following topics:\n",
+    "* [FastEstimator Search API](#searchapi)\n",
```

Comment:
the links need to be formatted like #ta12<text> since they need to be globally unique over all the tutorials (**T**utorial **A**dvanced **12** stuff) for proper rendering

Reply:
done